parley/src/layout/editor.rs

```diff
@@ -292,46 +311,45 @@ where
 
     /// Move to the next cluster left in visual order.
     pub fn move_left(&mut self) {
-        self.editor
-            .set_selection(self.editor.selection.previous_visual(
-                &self.editor.layout,
-                self.editor.cursor_mode,
-                false,
-            ));
+        self.editor.set_selection(
```

Comment:
This is a pre-existing concern, but I'll raise it now; should these methods call `refresh_layout`? That would be necessary if specifically the text has changed since the last relayout, and I don't think anything protects against that.

Reply:
I think I'll defer issues around this sort of thing to future work.